### PR TITLE
Add style to questions and fix logic for group single questions

### PR DESCRIPTION
**Pull request by polsieira**
## What does this PR do?

- [x] Fix styling on multiple questions
- [x] Add logic for singleGroup questions

### Where should the reviewer start?

- components/SingleGroupQ.js

#### How should this be manually tested?

- run `expo start` and confuse the questionnaire to see all types of questions

#### Have tests been written for all functionality?

- No

#### Screenshots

Screen Shot 2020-01-09 at 10 58 46 AM
Screen Shot 2020-01-09 at 10 58 42 AM
Screen Shot 2020-01-09 at 10 58 36 AM

### CODE REVIEW

File: components/GroupMultipleQ.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
+import React, { useState, Fragment } from 'react';
```
Comment (vrandall66) [inline components/GroupMultipleQ.js:1]: Glad this file made a comeback :smile:

File: components/GroupSingleQ.js [hunk 2]
```diff
@@ -89,9 +88,25 @@ export default function GroupSingleQ({ question, answerQuestion }) {
   });
 
   const handlePress = (id, index) => {
-    let IDs = [...checkWho];
-    IDs[index] = id;
-    setCheckWho(IDs);
+    let IDs;
```
Comment (vrandall66) [inline components/GroupSingleQ.js:91]: This logic is fantastic. Much better UE

File: screens/SplashScreen/SplashScreen.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
```
Comment (vrandall66) [inline screens/SplashScreen/SplashScreen.js:1]: Are we using this?

**Review by vrandall66 — APPROVED** [review]

**Comment by tylorschafer** [comment]
This all looks great. I don't think any of these changes would be affecting the calls to Infermedica.

**Pull request merged**